rosidl_generator_cpp/msg/PrimitivesConstants.msg

```diff
@@ -0,0 +1,14 @@
+bool BOOL_CONST=true
+byte BYTE_CONST=50
+char CHAR_CONST=100
+float32 FLOAT32_CONST=1.125
+float64 FLOAT64_CONST=3.14159
```

Comment:
For `float32` we selected a value which is representated exactly the same way (without rounding) in Python and C++. I guess this is not the case for this one. Even if it is only tested in C++ should we try to use a similar "portable" value?